packages/gatsby-theme-docs/src/pages/release-notes.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { MDXProvider } from '@mdx-js/react';
+import {
+  Markdown,
+  Subtitle,
+  ContentNotifications,
+} from '@commercetools-docs/ui-kit';
+import {
+  SEO,
+  Link,
+  ThemeProvider,
+  CodeExample,
+  MultiCodeExample,
+} from '../components';
+import PlaceholderMarkdownComponents from '../overrides/markdown-components';
+
+// See https://mdxjs.com/getting-started#table-of-components
+const components = {
```

Comment:
This shouldn't be duplicated. We should extract this and use it in both templates

Reply:
Thanks, but please note that the PR is still a draft.